SCons/Tool/MSCommon/vcTests.py

```diff
@@ -131,130 +133,78 @@ def runTest(self) -> None:
         except IOError as e:
             print("Failed trying to write :%s :%s" % (tools_version_file, e))
 
-
         # Test 14.3 (VS2022) and later
         vc_ge2022_list = SCons.Tool.MSCommon.vc._GE2022_HOST_TARGET_CFG.all_pairs
-
         for host, target in vc_ge2022_list:
             batfile, clpathcomps = SCons.Tool.MSCommon.vc._GE2022_HOST_TARGET_BATCHFILE_CLPATHCOMPS[(host,target)]
             # print("GE 14.3 Got: (%s, %s) -> (%s, %s)"%(host,target,batfile,clpathcomps))
 
             env={'TARGET_ARCH':target, 'HOST_ARCH':host}
+            path = os.path.join('.', "Auxiliary", "Build", batfile)
+            MSVcTestCase._createDummyFile(path, batfile, add_bin=False)
             path = os.path.join('.', 'Tools', 'MSVC', MS_TOOLS_VERSION, *clpathcomps)
-            MSVcTestCase._createDummyCl(path, add_bin=False)
+            MSVcTestCase._createDummyFile(path, 'cl.exe', add_bin=False)
             result=check(env, '.', '14.3')
             # print("for:(%s, %s) got :%s"%(host, target, result))
             self.assertTrue(result, "Checking host: %s target: %s" % (host, target))
 
-        # Now test bogus value for HOST_ARCH
```

Comment:
Why remove this test?

Reply:
Redundant tests were removed.

The two tests in question are:
1) `env={'TARGET_ARCH':'x86', 'HOST_ARCH':'GARBAGE'}`
2) `env={'TARGET_ARCH':'GARBAGE', 'HOST_ARCH':'x86'}`

These two tests are *msvc version independent* as long as `x86` is a supported target.

There is no reason to perform the tests for each "block" of msvc versions.  Redundant tests were removed.

These two tests still exist and are the last two tests of this function (formerly "attached" for msvc version 6.0).  Old lines 254-274, new lines 204-224.

The path was changed from `6.0` to `14.3` should older versions be removed in the future.